cypress/integration/onboarding/createWallet.spec.ts

```diff
@@ -63,6 +63,12 @@ describe('Create wallet on first visit', () => {
       cy.contains('Next').click()
     })
   })
+  it('sets preferences', () => {
+    cy.get('.MuiBox-root > span').should('have.class', 'MuiSwitch-root').click({ multiple: true })
+    cy.get('.MuiButton-outlined').contains('USD').dblclick()
```

Comment:
@calvinkei tell cypress to double click the dropdown component instead of selecting one item from the list for the time being.